client/context/query.go

```diff
@@ -31,24 +31,33 @@ func (ctx CLIContext) GetNode() (rpcclient.Client, error) {
 
 // Query performs a query for information about the connected node.
 func (ctx CLIContext) Query(path string, data cmn.HexBytes) (res []byte, err error) {
-	return ctx.query(path, data)
+	resp, err := ctx.query(path, data)
+	return resp.Value, err
 }
 
 // Query information about the connected node with a data payload
 func (ctx CLIContext) QueryWithData(path string, data []byte) (res []byte, err error) {
-	return ctx.query(path, data)
+	resp, err := ctx.query(path, data)
+	return resp.Value, err
 }
 
 // QueryStore performs a query from a Tendermint node with the provided key and
 // store name.
 func (ctx CLIContext) QueryStore(key cmn.HexBytes, storeName string) (res []byte, err error) {
+	res, _, err = ctx.queryStore(key, storeName, "key")
+	return res, err
+}
+
+// QueryStoreWithHeight performs a query from a Tendermint node with the provided key and
+// store name. Returns the queried data along with the height the query was performed at.
+func (ctx CLIContext) QueryStoreWithHeight(key cmn.HexBytes, storeName string) (res []byte, height int64, err error) {
 	return ctx.queryStore(key, storeName, "key")
 }
 
 // QuerySubspace performs a query from a Tendermint node with the provided
 // store name and subspace.
 func (ctx CLIContext) QuerySubspace(subspace []byte, storeName string) (res []sdk.KVPair, err error) {
-	resRaw, err := ctx.queryStore(subspace, storeName, "subspace")
+	resRaw, _, err := ctx.queryStore(subspace, storeName, "subspace")
```

Comment:
thanks! we need to fix this everywhere on the codebase

Reply:
please open an issue for this work!